src/Psr/Tracing/RelayOpenTelemetry.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CacheWerk\Relay\Psr\Tracing;
+
+use Relay\Relay;
+
+use OpenTelemetry\API\Trace\SpanKind;
+use OpenTelemetry\API\Trace\TracerInterface;
+
+class RelayOpenTelemetry
+{
+    /**
+     * The Relay instance.
+     *
+     * @var \Relay\Relay
+     */
+    protected Relay $relay;
+
+    /**
+     * The OpenTelemetry tracer instance.
+     *
+     * @var \OpenTelemetry\API\Trace\TracerInterface
+     */
+    protected TracerInterface $tracer;
+
+    /**
+     * Creates a new `NewRelicRelay` instance.
+     *
+     * @param  \Relay\Relay  $relay
+     * @param  \OpenTelemetry\API\Trace\TracerInterface  $tracer
+     * @return void
+     */
+    public function __construct(Relay $relay, TracerInterface $tracer)
+    {
+        $this->relay = new $relay;
+        $this->tracer = new $tracer;
+    }
+
+    /**
+     * Executes Relay methods inside OpenTelemetry span.
+     *
+     * @param  string  $name
+     * @param  array  $arguments
+     * @return mixed
+     */
+    public function __call(string $name, array $arguments)
+    {
+        $span = $this->tracer->spanBuilder('Relay') // TODO: What's the name?
+            ->setAttribute('command', strtolower($name))
```

Comment:
I would remove `command` since it is the same as span name and instead you should add `db.statement` that contains both `$name` and `$arguments`. I guess Relay is still Redis so this official [example](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#redis) should apply here.

Reply:
I added `db.operation` instead. This link was helpful.